quantum-api/test/files/fileTransform.spec.um

```diff
@@ -37,8 +37,9 @@
     @changelogList
       @changelog 0.1.0
         @description: Version 0.1.0
-        @change added
-          @description: Api added
+        @entry
+          @change added
+            @description: Api added
```

Comment:
Changes don't display in the changelog unless they are inside an `entry` - Updated the transform to move changes directly inside an `api` block inside an `entry`